# Path to the folder that has the frozen_inference_graph.pb file in it
#graph_path: "~/networks/ssd_mobilenet_v1.pb"
#graph_path: "~/networks/ssd_inception_v2.pb"
#graph_path: "~/networks/faster_rcnn_resnet101.pb"
#graph_path: "~/networks/rfcn_resnet101.pb"

graph_path: "~/networks/ssd_mobilenet_v1_laptop.pb"
#graph_path: "~/networks/ssd_mobilenet_v1_laptop.tflite"

# File with all the label names in it
#labels_path: "~/networks/tf_label_map.pbtxt"
labels_path: "~/networks/labels_laptop.txt"

# Topic to get the images from
camera_namespace: "/camera/rgb/image_rect_color"

# Threshold for what we'll decide is a real object
threshold: 0.50

# Maximum amount of system memory to use in TensorFlow, in range 0 - 1
memory: 0.2

# Output debug images
debugImage: true
